### PR TITLE
add sendMessage function

### PR DESCRIPTION
**Pull request by bartacruz**

### CODE REVIEW

**Comment by bartacruz** [comment]
Sorry, I've updated my fork with an old testing version. That's why I've closed the PR.
Now it's ok.

**Comment by stewones** [comment]
yo @bartacruz, can you check that conflicting file?

**Comment by bartacruz** [comment]
Hi @stewones, it's done. Please let me know if there is anything else I can do

File: android/src/main/java/com/getcapacitor/community/fcm/FCMPlugin.java [hunk 1]
```diff
@@ -77,12 +84,12 @@ public void refreshToken(final PluginCall call) {
         FirebaseMessaging.getInstance().deleteToken();
         FirebaseInstallations.getInstance().getToken(false).addOnSuccessListener(getActivity(), instanceIdResult -> {
             JSObject data = new JSObject();
-            data.put("token", instanceIdResult.getToken());
+            data.put("token", tokenResult.getResult());
```
Comment (stewones) [inline android/src/main/java/com/getcapacitor/community/fcm/FCMPlugin.java:87]: @bartacruz is this line really correct?

Comment (stewones) [reply android/src/main/java/com/getcapacitor/community/fcm/FCMPlugin.java:87]: you're expecting `instanceIdResult` there's no `tokenResult` callback on this method

**Review by stewones — CHANGES_REQUESTED** [review]
thanks for the PR, it looks good to me just want to clarify the question above.

**Comment by stewones** [comment]
almost 2 years now, closing due to inactivity.

**Pull request closed without merging**